opendrift/readers/reader_telemac_selafin.py

```diff
@@ -0,0 +1,263 @@
+import numpy as np
+from datetime import datetime, timedelta
+import sys
+from os import path, environ, sep
+from scipy.spatial import cKDTree
+#Activating PYTEL
+dir = environ['HOMETEL']+'{}scripts{}python3'.format(sep,sep)
+sys.path.append(path.join( path.dirname(sys.argv[0]),dir))#pytel path use environment variables
+from data_manip.formats.selafin import Selafin
```

Comment:
I believe this line can be added to environment.yml under the pip section for automatic installation of this package:
```git://github.com/CNR-Engineering/PyTelTools.git#egg=pyteltools```

Reply:
I am not sure pyteltools is actually developed by the Telemac team. It is unclear if the two projects are linked. I can ask